js/whitebit.js

```diff
@@ -1188,13 +1176,19 @@ module.exports = class whitebit extends Exchange {
             } else {
                 // stop market order
                 method = 'v4PrivatePostOrderStopMarket';
+                if (marginMode !== undefined || marketType === 'swap') {
```

Comment:
We're repeating here
`marginMode !== undefined || marketType === 'swap'` 
and 
`marginMode !== undefined && marginMode !== 'cross'`

three times each, in the future, if we need to update this, we need to do it in three places. Please try to dry the code next time.